src/Microsoft.AspNetCore.Antiforgery/Internal/AntiforgeryOptionsSetup.cs

```diff
@@ -28,7 +28,14 @@ public static void ConfigureOptions(AntiforgeryOptions options, DataProtectionOp
 
         private static string ComputeCookieName(string applicationId)
         {
-            using (var sha256 = SHA256.Create())
+            HashAlgorithm sha256;
+#if NETSTANDARD1_3
+            sha256 = SHA256.Create();
+#else
+            sha256 = new SHA256Cng();
```

Comment:
Wasn't `(HashAlgorithm)new SHA256CryptoServiceProvider()` recommended in one of the (zillion) #95 comments? Not sure of the advantages / disadvantages but wanted to check.